5.6/apache/Dockerfile

```diff
@@ -50,5 +50,5 @@ RUN { \
     && a2enmod rewrite
 
 # Install composer
-COPY install-composer /install-composer
+COPY image-files /
```

Comment:
You won't get less layers than one for all possible files we need to add - which we have with this method.

> We hardly ever have to change it anymore.

I always doubt that, there are already 2 ones as you mentioned. Looks like this should be changed in composer ;)